rllib/contrib/maddpg/maddpg_policy.py

```diff
@@ -130,8 +130,9 @@ def _make_ph_n(space_n, name=""):
             new_act_ph_n,
             obs_space_n,
             act_space_n,
-            hiddens=config["critic_hiddens"],
-            activation=getattr(tf.nn, config["critic_hidden_activation"]),
+            config["use_state_preprocessor"],
```

Comment:
In the future, prefer to pass named args with the name for readability.

Reply:
Yes, and we should start using *s in the method signature (only py3 supports those) to enforce using named args after maybe the initial 1-2 first arguments (which are often a given and maybe don't need to be named).